sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryTests.cs

```diff
@@ -37,19 +41,40 @@ public async Task Query_ValidQuery_Success()
                 // Create a room twin, with property "IsOccupied": true
                 string roomTwinId = await GetUniqueTwinIdAsync(client, TestAssetDefaults.RoomTwinIdPrefix).ConfigureAwait(false);
                 BasicDigitalTwin roomTwin = TestAssetsHelper.GetRoomTwinPayload(roomModelId);
-                await client.CreateOrReplaceDigitalTwinAsync<BasicDigitalTwin>(roomTwinId, roomTwin).ConfigureAwait(false);
+                await client.CreateOrReplaceDigitalTwinAsync(roomTwinId, roomTwin).ConfigureAwait(false);
 
-                string queryString = "SELECT * FROM digitaltwins where IsOccupied = true";
+                // Construct a query string to find the twins with the EXACT model id and provided version. If EXACT is not specified, the query
+                // call will get all twins with the same model id but that implement any version higher than the provided version
+                string queryString = $"SELECT * FROM digitaltwins where " +
+                                    $"IS_OF_MODEL('{roomModelId}', EXACT) " +
```

Comment:
This isn't indented in a way that makes sense to me.

It would either just be 4 spaces in from the parent line, or inline with the SQL statement above (so in 1 more space).

Reply:
Also `where` is a keyword, that doesn't get its own line, isn't capitalized like the other keywords (although not necessary, not consistent)